### DISCUSSION TITLE
Unable to run the autodrive simulator

### DISCUSSION THREAD

**Discussion by SumeetShanbhag** [Tinker-Twins/AutoDRIVE#7] [Q&A] [answered]
Hello, I was trying to run the multi agent scenario simulator but after trying several different things, I wasn't able to do so. I enabled all permissions for the files present inside the folder using "sudo chmod +x 'file_name' ". Can you tell me what could be the issue and how to resolve it?

**Comment by Tinker-Twins** [comment, marked as answer]
Hey Sumeet,

Try uptating your graphics drivers (if possible) and see if it helps. The newer version of AutoDRIVE Simulator supports Vulkan Graphics API, which is supported by later versions of NVIDIA drivers. Hope this helps!

Best,

**Reply by Will70330** [reply]
Is vulkan the only supported driver for 0.3.0? or is openGL still supported? 

seeing issues with executing on a Parallels VM with 22.04 running on an older intel MacBook. Pretty sure Parallels is using openGL under the hood with no support for Vulkan (just from some preliminary searching).

Looking for any support on this if possible.

**Reply by Tinker-Twins** [reply]
Hey,

Unfortunately, Unity HDRP has depreciated support for OpenGL or OpenGL ES devices. You can read more about the system requirements [here](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@17.0/manual/System-Requirements.html).

Consequently, AutoDRIVE Simulator, which is developed atop Unity HDRP, is not natively supported on OpenGL or OpenGL ES devices.

That being said, if you have some experience with Unity yourself, you can download the source code from GitHub and open the project using a the built-in rendering pipeline, which still supports OpenGL Graphics API. Alternatively, opening with an earlier versions of Unity (e.g. 2018) and corresponding HDRP packages can help. There are also readily available assets on the Unity Asset Store to help with this downgrade.

We are also working on this front and will update you when it is accomplished.

Hopefully this gets you started with AutoDRIVE soon!
